### ISSUE TITLE
Build failure: davinci-resolve

### ISSUE THREAD

**Issue by gnumber13** [NixOS/nixpkgs#244049]
### Steps To Reproduce

Try to build (temporarily to try it out) *davinci-resolve* with `export NIXPKGS_ALLOW_UNFREE=1 && nix-shell -p davinci-resolve`

### Build log
Executed ` nix-store -l /nix/store/n8pclg9cn96010qb0wcq16yc83s5fsy3-davinci-resolve-src.zip.drv` and got:
```
curl: (3) URL rejected: Malformed input to a URL function
```

### Notify maintainers
@jshcmpbll
@yuuyins 

### Metadata
```console
 - system: `"x86_64-linux"`
 - host os: `Linux 6.3.9-200.fsync.fc38.x86_64, Nobara Linux, 38 (Thirty Eight), nobuild`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.16.1`
 - channels(root): `"nixpkgs"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixpkgs`
```

**Comment by jshcmpbll** [comment]
This should resolve it:

https://github.com/NixOS/nixpkgs/pull/242751

**Comment by gnumber13** [comment]
Thank you I'll keep an eye out!
And sorry this seems to be a duplicate of #242091 if I'm not wrong.
Shall I close the issue?

**Comment by jshcmpbll** [comment]
No worries, yeah lets close this one so anyone else looking references that issue.

You can always build off my working branch for now if you need the tool but hopefully I get a review and merge soon on my PR

**Comment by gnumber13** [comment]
Thanks. Good to know.
Alright. This is issue is now closed due to beeing a duplicate issue to #242091.

**Comment by gnumber13** [comment]
Thanks. Good to know.
Alright. This is issue is now closed due to beeing a duplicate issue to https://github.com/NixOS/nixpkgs/issues/242091.
